test/integration/features/cors.feature

```diff
@@ -0,0 +1,23 @@
+@cors
+Feature: Cors Add-on 
+  As Minishift user I can install and apply Cors add-on from minishift-addons repository.
+
+  Scenario: Installing the add-on
+     When file from "https://raw.githubusercontent.com/minishift/minishift-addons/master/add-ons/cors/cors.addon" is downloaded into location "download/cors"
+      And executing "minishift addons install ../../testing/integration-test/download/cors" succeeds
+     Then stdout should contain "Addon 'cors' installed"
+
+  Scenario: Minishift starts
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Applying the add-on
+    Given Minishift has state "Running"
+     When executing "minishift addons apply cors" succeeds
+     Then stdout should contain "CORS is now allowed from any address"
+
+  Scenario: Deleting Minishift
+    Given Minishift has state "Running"
+     When executing "minishift delete --force" succeeds
+     Then Minishift should have state "Does Not Exist"
```

Comment:
@agajdosi Ideally we should should have an example where we can apply, remove, uninstall the add-on. But maybe it is  another task.

Reply:
@LalatenduMohanty I would like to have the example in this PR. Ideally it should work I think, so I took the cors add-on, but it can't be removed. Wondering if there is a better candidate for this or if I could implement the `cors.addon.remove`. Then we could just mention it as an example.

Other plan could be to create an example addon, which would do nothing just install, apply, remove and uninstall itself. This file could be used for the add-on documentation also because it would contain all the needed basics which add-ons need. After that I can create an `example.feature` which would show the minimal setup of the feature file to test minimal functionality of add-on.

The second seems fine to me, the question is whether @minishift/minishift-dev are fine with it?

Reply:
> The second seems fine to me, the question is whether @minishift/minishift-dev are fine with it?

+1 for this. `cors` is certainly not the good example. We can add it as part of another PR where we need to add tests.

Reply:
@budhrg @LalatenduMohanty I have added `example.addon` which contains just `echo` and then `example.feature` which tests this blank addon. It is also used in documentation.

Please note that the example feature has two variant: `example.feature` and `example-clean.feature`, the second one is less commented and I think looks more tidy and readable. I plan we choose and use only one of these variants.